server/src/test/java/org/opensearch/gateway/remote/RemoteClusterStateServiceTests.java

```diff
@@ -186,6 +201,201 @@ public void testWriteIncrementalMetadataSuccess() throws IOException {
         assertThat(manifest.getStateUUID(), is(expectedManifest.getStateUUID()));
     }
 
+    public void testReadLatestMetadataMarkerFailedIOException() throws IOException {
```

Comment:
Marker -> Manifest, Applies at all the places

Reply:
done